android/java/res/layout/fragment_setup_wallet.xml

```diff
@@ -15,80 +15,178 @@
-            android:clickable="true"
-            android:focusable="true"
-            android:textAllCaps="false"
+            android:layout_marginHorizontal="32dp"
             android:layout_marginBottom="16dp"
-            android:text="@string/restore"
-            android:textSize="16sp"
-            android:textColor="@color/wallet_text_color"/>
+            android:foreground="?android:attr/selectableItemBackground"
+            android:clickable="true"
+            app:cardBackgroundColor="@color/brave_wallet_onboarding_card_view_background"
+            app:cardCornerRadius="10dp"
+            app:cardElevation="2dp"
+            app:cardUseCompatPadding="true">
+
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_margin="32dp">
+                <ImageView
+                    android:id="@+id/restore_wallet_icon"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:contentDescription="@null"
+                    android:src="@drawable/restore_wallet"
+                    android:layout_marginEnd="16dp"
+                    android:layout_alignParentTop="true"
+                    android:layout_alignParentStart="true"/>
+
+                <TextView
+                    android:id="@+id/restore_text_view"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="8dp"
+                    android:text="@string/already_have_a_wallet"
+                    android:textColor="@color/wallet_text_color"
+                    android:textSize="22sp"
+                    android:lineSpacingExtra="2sp"
+                    android:layout_toEndOf="@+id/restore_wallet_icon"
+                    style="@style/BraveWalletTextViewTitle"
+                    />
+
+                <TextView
+                    android:id="@+id/restore_description_text_view"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="8dp"
+                    android:text="@string/already_have_a_wallet_description"
+                    android:textColor="@color/wallet_onboarding_secondary_text_color"
+                    android:textSize="14sp"
+                    android:lineSpacingExtra="4sp"
+                    android:layout_toEndOf="@+id/restore_wallet_icon"
+                    android:layout_below="@+id/restore_text_view"
+                    />
+
+                <ImageView
+                    android:id="@+id/brave_icon"
+                    android:layout_width="24dp"
+                    android:layout_height="24dp"
+                    android:contentDescription="@null"
+                    android:src="@drawable/ic_brave"
```

Comment:
I would check all `src` if we want to go with `srcCompat`

Reply:
If you have a preference over `srcCompat` I'm happy to change all the occurrences.

Reply:
no, I'm good, just wanted to be extra careful that we can use it now. I'm totally fine to have `src`.